src/Fixer/StringNotation/StringLineEndingFixer.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\StringNotation;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\Fixer\WhitespacesAwareFixerInterface;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * Fixes the line endings in multi-line strings.
+ *
+ * @author Ilija Tovilo <ilija.tovilo@me.com>
+ */
+final class StringLineEndingFixer extends AbstractFixer implements WhitespacesAwareFixerInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
+    {
+        return true;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isRisky()
+    {
+        return true;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefinition()
+    {
+        return new FixerDefinition(
+            'All multi-line strings must use same line ending.',
+            [
+                new CodeSample(
+                    "<?php \$a = 'my\r\nmulti\nline\r\nstring';\r\n"
+                ),
+            ],
+            null,
+            'Changing the line endings of multi-line strings might change behaviour.'
```

Comment:
Basically, you said here:
`sprintf("Doing %s might change behaviour.", $description)`.
I don't see any value here, change behaviour of what? in what way?

Reply:
@keradus Well, for one if you store the string anywhere, different line endings will be stored. String comparison could also potentially fail. Do you have a suggestion?

Reply:
I do know that this fixer is risky.
Yet, this line is suppose to describe **why** it's risky, and you are just copy-pasting description only

Reply:
Ok, what about:

> Changing the line endings of multi-line strings might change string comparisons and outputs.

Of something along those lines.